server/src/main/java/org/opensearch/search/backpressure/trackers/CpuUsageTracker.java

```diff
@@ -78,18 +63,10 @@ public Optional<TaskCancellation.Reason> checkAndMaybeGetCancellationReason(Task
         );
     }
 
-    public long getCpuTimeNanosThreshold() {
-        return TimeUnit.MILLISECONDS.toNanos(cpuTimeMillisThreshold);
-    }
-
-    public void setCpuTimeMillisThreshold(long cpuTimeMillisThreshold) {
-        this.cpuTimeMillisThreshold = cpuTimeMillisThreshold;
-    }
-
     @Override
-    public TaskResourceUsageTracker.Stats stats(List<? extends Task> activeTasks) {
-        long currentMax = activeTasks.stream().mapToLong(t -> t.getTotalResourceStats().getCpuTimeInNanos()).max().orElse(0);
-        long currentAvg = (long) activeTasks.stream().mapToLong(t -> t.getTotalResourceStats().getCpuTimeInNanos()).average().orElse(0);
+    public TaskResourceUsageTracker.Stats stats(List<? extends Task> tasks) {
```

Comment:
nitpick: Calling it `activeTasks` in the TaskResourceUsageTracker interface was deliberate as only the ongoing tasks are passed to this method. I feel that renaming it to `tasks` loses that context.

Reply:
Got it. Updated in all the trackers.